streams/ve.m3u

```diff
@@ -91,25 +87,29 @@ https://vcp2.myplaytv.com/tvfanb/tvfanb/playlist.m3u8
 #EXTINF:-1 tvg-id="TVS.ve",TVS (720p)
 https://vcp3.myplaytv.com/tvs/tvs/playlist.m3u8
 #EXTINF:-1 tvg-id="ValeTV.ve",Vale TV (480p) [Not 24/7]
-https://vcp2.myplaytv.com/valetv/valetv/master.m3u8
-#EXTINF:-1 tvg-id="ValeTV.ve",Vale TV (480p)
+https://cloud.fastchannel.es/manifiest/hls/valetv/valetv_480p.m3u8
+#EXTINF:-1 tvg-id="ValeTV.ve",Vale TV (480p) [Not 24/7]
 https://vcp2.myplaytv.com/valetv/valetv/playlist.m3u8
-#EXTINF:-1 tvg-id="Venevision.ve",Venevisión (480p)
-http://wwz.cisneros.tv/ch-estudio4/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="VenezolanadeTelevision.ve",Venezolana de Televisión (720p)
+#EXTINF:-1 tvg-id="Venevision.ve",Venevisión (720p)
+https://vod2live.univtec.com/manifest/4c41c0d8-e2e4-43cc-bd43-79afe715e1b3.m3u8
+#EXTINF:-1 tvg-id="Venevision.ve",Venevisión (1080p)
+https://venevision.akamaized.net/hls/live/2098814/VENEVISION/master.m3u8
+#EXTINF:-1 tvg-id="VenezolanadeTelevision.ve",Venezolana de Televisión (720p) [Not 24/7]
 #EXTVLCOPT:http-referrer=https://odysee.com/
 https://cdn.odysee.live/content/9ad6390b9047d4c43ea0ae6ca3199a3ef70fc8aa/master.m3u8
 #EXTINF:-1 tvg-id="VepacoTV.ve",Vepaco TV (480p)
 https://cloud.streamingconnect.tv/hls/vepacotv/vepacotv.m3u8
 #EXTINF:-1 tvg-id="VePlus.ve",Ve Plus (480p) [Not 24/7]
 http://190.122.96.187:8888/http/006
```

Comment:
I do not know if it's an national or international stream, but this Ve Plus stream differs from the official Venevision Play one.

Reply:
From Wikipedia

_VePlus operates on three different feeds:

VePlus: Broadcasts in Latin America and [Spain](https://en.wikipedia.org/wiki/Spain), excluding Venezuela.
[VePlus](https://en.wikipedia.org/wiki/Venevisi%C3%B3n_Plus): Exclusive feed for Venezuela.
VePlus: Available in the [United States](https://en.wikipedia.org/wiki/United_States)._

Reply:
> From Wikipedia:
> 
> VePlus operates on three different feeds:
> 
> VePlus: Broadcasts in Latin America and [Spain](https://en.wikipedia.org/wiki/Spain), excluding Venezuela. [VePlus](https://en.wikipedia.org/wiki/Venevisi%C3%B3n_Plus): Exclusive feed for Venezuela. VePlus: Available in the [United States](https://en.wikipedia.org/wiki/United_States).

Interesting, thank you for the addition. Then this is a separate stream. Most certainly a US-exclusive stream.